mock-mhs-adaptor/src/main/java/uk/nhs/adaptors/mockmhsservice/producer/InboundProducer.java

```diff
@@ -0,0 +1,19 @@
+package uk.nhs.adaptors.mockmhsservice.producer;
+
+import org.springframework.jms.JmsException;
+import org.springframework.jms.core.JmsTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class InboundProducer {
```

Comment:
Rename to inboundDispatcher

Reply:
eh InboundProducer is fine. "Producer" is the name of the entity putting messages onto a queue. "Consumer" is the entity that reads them.